composer.json

```diff
@@ -39,5 +39,12 @@
         "branch-alias": {
             "dev-master": "1.4-dev"
         }
+    },
+    "scripts": {
+        "cs-fix-diff": "php-cs-fixer fix --dry-run --diff",
+        "cs-fix": "php-cs-fixer fix"
+    },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^3.49"
```

Comment:
what about using `phive` to install it for local development? (or use https://github.com/PHP-CS-Fixer/shim phar) this would reduce the dependencies IMO

Reply:
Agreed, to have php-cs-fixer as independent binary.